l2-contracts/contracts/ConsensusRegistry.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.20;
+
+import {EfficientCall} from "@matterlabs/zksync-contracts/l2/system-contracts/libraries/EfficientCall.sol";
+
+/// @title ConsensusRegistry
+/// @dev A contract to manage consensus nodes and committees.
+contract ConsensusRegistry {
+    // Owner of the contract (i.e., the governance contract).
+    address public owner;
+    // An array to keep track of node owners.
+    address[] public nodeOwners;
+    // A map of node owners => nodes
+    mapping(address => Node) public nodes;
+    // The current validator committee list.
+    CommitteeValidator[] public validatorCommittee;
+    // The current attester committee list.
+    CommitteeAttester[] public attesterCommittee;
+
+    /// @dev Represents a consensus node.
+    struct Node {
+        // A flag stating if the node is inactive.
+        // Inactive nodes are not considered when selecting committees.
+        bool isInactive;
```

Comment:
why double negative? `isActive` sounds easier to understand + storage has `false` value by default, so in case you access the non-existing node it will have the default status as not active.

Reply:
Agree, it made a slight change in the early design, but no longer. Changes here: https://github.com/matter-labs/era-contracts/pull/555/commits/c4069e38c1a425fc39f54e52a0b33a502dbaf9c5